pkg/trace/config/config.go

```diff
@@ -311,8 +311,9 @@ type AgentConfig struct {
 	Endpoints []*Endpoint
 
 	// Concentrator
-	BucketInterval   time.Duration // the size of our pre-aggregation per bucket
-	ExtraAggregators []string
```

Comment:
Should `ExtraAggregators` get marked as deprecated as opposed to deleted? My worry here is that it might still live in a customer configuration somewhere, and if we were to re-introduce it in the future we could run into problems. I think this has to be considered like a "reserved" field of sorts.

Reply:
Alternatively, we could start using is again with `peer.service` as a possible value that we set if enabled.

Reply:
I've added this back with a comment about it being deprecated. It's not actually used in the current agent code because we don't create aggregation keys with strings anymore. We have the new struct key format, so we can't arbitrarily aggregate over other tag keys.

I think this only works for much older versions of the agent (probably any agent that doesn't use the new ClientStatsPayloads).